.github/workflows/post_release_version_bump.yml

```diff
@@ -12,21 +12,12 @@ jobs:
       - name: Clone repository
         uses: actions/checkout@v2
 
-      - name: Install Python
-        uses: actions/setup-python@v2
+      - name: Run post-release version bump
+        uses: PennyLaneAI/automation/version_bump_action@v1.0
         with:
-          python-version: '3.9'
-
-      - name: Install dependencies
-        run: |
-          python -m pip install --upgrade pip
-          pip install pennylane
-
-      - name: Run version bump
-        run: >
-          python .github/workflows/vb_script.py
-          --version_path "./pennylane_sf/_version.py"
-          --changelog_path "./.github/CHANGELOG.md" --post_release
+          version_path: "./pennylane_sf/_version.py"
+          changelog_path: "./.github/CHANGELOG.md"
```

Comment:
Oh, it's great we can input this! :open_mouth: :100:

Reply:
Yes! we can set inputs and extract outputs as well! In this way a composite action can be run, and then outputs extracted and passed to local actions ! Its a very flexible framework!